lib/offsite_payments/integrations/molpay.rb

```diff
@@ -176,6 +183,10 @@ def initialize(query_string, options = {})
         def success?
           @notification.acknowledge
         end
+        
+        def pending?
+          params['status'] == "22"
```

Comment:
Should this use the new `status` method? Then we'll only be parsing `params['status']` in one place.
```
def pending?
  status == 'Pending'
end
```